test/test_browser.py

```diff
@@ -4792,7 +4792,7 @@ def test_webgl_simple_extensions(self, simple_enable_extensions, webgl_version):
     self.btest_exit('webgl2_simple_enable_extensions.c', args=cmd)
 
   @parameterized({
-    '': ([],),
+    'default': ([],),
```

Comment:
Any reason you prefer this?

Reply:
Oh right. I did this because I totally forgot that other variants of this test existed, and this would have reminded me to run `browser.test_webgpu_basic_rendering_*` instead.

Reply:
I see.  Fair enough.  I tent to prefer the suffixed form for the default case.  One reason is that I like to match the name of the test itself with the name of a file on disc.  Not a big deal though.